config/audit/utility/java.yaml

```diff
@@ -0,0 +1,45 @@
+auditConfig:
+  - id: AuditUtility.ApacheCommons
+    name: Apache Commons
+    patterns:
+      - '^org.apache.commons.*:commons.*'
+
+  - id: AuditUtility.GoogleGuava
+    name: Google guava
+    patterns:
+      - '^com.google.guava.*:guava.*'
+
+  - id: AuditUtility.CommonsIo
+    name: Commons Io
+    patterns:
+      - '^commons-io.*:commons-io.*'
+
+  - id: AuditUtility.ApacheLogging
+    name: Apache Logging Log4j
+    patterns:
+      - '^org.apache.logging.log4j.*:log4j-core.*'
```

Comment:
These are leakage libraries. These will not come under utility libraries

Reply:
okay removing it.